rln/src/protocol.rs

```diff
@@ -53,6 +58,98 @@ pub struct RLNProofValues {
     rln_identifier: Field,
 }
 
+pub fn serialize_witness(rln_witness: RLNWitnessInput) -> Vec<u8> {
```

Comment:
Here, we could integrate that with Serde instead of writing a "stand-alone" function.